test/unit/resources/cpan_test.rb

```diff
@@ -0,0 +1,15 @@
+# encoding: utf-8
+# author: Christoph Hartmann
+# author: Dominik Richter
+
+require 'helper'
+require 'inspec/resource'
+
+describe 'Inspec::Resources::Cpan' do
+  it 'verify cpan package detail parsing' do
+    resource = load_resource('cpan', 'DBD::Pg')
+      pkg = {:type=>'cpan', :name=>'DBD::Pg', :version=>'3.7.0', :installed=>true }
```

Comment:
Looks like indentation is off here. Also, you can write this with the newer non-hash-rocket hash syntax:

```ruby
pkg = {type: 'cpan', name: 'DBD::Pg', version: '3.7.0', installed: true}
```